block/submit.go

```diff
@@ -158,6 +158,11 @@ func (m *Manager) CreateAndSubmitBatch(maxSizeBytes uint64) (*types.Batch, error
 	if err != nil {
 		return nil, fmt.Errorf("create batch: %w", err)
 	}
+	// This is the last batch, so we need to mark it as such
+	if lastBatch && b.EndHeight() == endHeightInclusive {
+		b.LastBatch = true
+		m.logger.Debug("Last batch created.", "start height", startHeight, "end height", endHeightInclusive)
+	}
 
 	m.logger.Info("Created batch.", "start height", startHeight, "end height", endHeightInclusive)
```

Comment:
instead of having two logs you can put last as data in the existing log

Reply:
yeah i know, thought it will look better in the logs
fixed